tests/common/aws_test_framework.c

```diff
@@ -80,7 +80,7 @@ void TEST_NotifyTestStart()
 {
     /* Wait for test script to open serial port before starting tests on the
      * board.*/
-    vTaskDelay( pdMS_TO_TICKS( 5000UL ) );
+    vTaskDelay( pdMS_TO_TICKS( 10000UL ) );
```

Comment:
Would we consider something like this that takes a -D that is only included by the test infra build? https://github.com/aws/amazon-freertos/commit/7b43e18e667eef235fd991059b7d849403eff8b2

Reply:
Closed.